WMIAdventure/frontend/src/js/pages/creative/CardsCreator/molecules/EffectChoose/EffectChoose.js

```diff
@@ -40,6 +40,10 @@ class EffectChoose extends React.Component {
                                           tooltip={effect.tooltip}
                                           rank={1}
                                           effect={effect}
+                                          chosen={
+                                              // If chosenEffects list contains current effect then mark it as chosen
+                                              !!this.props.chosenEffects[0].find(x => x.id === effect.id)

```

Comment:
Dlaczego tutaj dwa wykrzykniki? Bez wykrzykników chyba też będzie działać z tego co widzę.

Reply:
Niestety nie, pierwszy wykrzyknik to negacja, a drugi sprawia, że to wyrażenie zwraca boolean. WebStorm sam mi pomógł wygenerować te dwa wykrzykniki.

Reply:
Próbowałem w konsoli też bez wykrzykników, ale musiały być żeby to wyrażenie zwracało boolean

Reply:
Ok, to jak tak musi być to spoko